data-prepper-core/src/test/java/org/opensearch/dataprepper/plugin/DefaultPluginFactoryTest.java

```diff
@@ -46,6 +51,7 @@ class DefaultPluginFactoryTest {
     private PluginSetting pluginSetting;
     private PluginBeanFactoryProvider beanFactoryProvider;
     private BeanFactory beanFactory;
+    private static final String TEST_PIPELINE_NAME = UUID.randomUUID().toString();
```

Comment:
This should just be a field and generated in `@BeforeEach`.

Reply:
Added